waltz-tools/src/main/java/com/wepay/waltz/tools/performance/PerformanceCli.java

```diff
@@ -65,11 +67,14 @@ private PerformanceCli(String[] args, boolean useByTest) {
         private CountDownLatch allClientsReady;
         private CountDownLatch allMountComplete;
 
+        private Map<Integer, Long> partitionHighWaterMark;
```

Comment:
NIT: rename the variable to `partitionHighWaterMarkMap` may be? because otherwise it feels that the variable is referring to a Long value.

Reply:
Why are we defining this variable at a Class level? It can be declared as a local variable to the processCmd(..) right?